comps/third_parties/wav2lip/src/Dockerfile.intel_hpu

```diff
@@ -22,16 +22,16 @@ COPY comps /home/user/comps
 COPY comps/third_parties/wav2lip/src/entrypoint.sh /usr/local/bin/entrypoint.sh
 
 # Install ffmpeg with x264 software codec
-RUN git clone https://github.com/FFmpeg/FFmpeg.git /home/user/comps/animation/FFmpeg
-WORKDIR /home/user/comps/animation/FFmpeg
-RUN ./configure --enable-gpl --enable-libx264 && \
-    make -j$(nproc-1) && \
+RUN git clone --depth 1 --branch ${REPO_VER} https://github.com/FFmpeg/FFmpeg.git /home/user/comps/animation/FFmpeg && \
+    cd /home/user/comps/animation/FFmpeg && \
+    ./configure --enable-gpl --enable-libx264 --enable-cross-compile --disable-debug --disable-doc --disable-ffprobe --disable-ffplay && \
```

Comment:
Why  the 2 FFmpeg Dockerfiles are configured with `--enable-cross-compile`?   Surely this is not compiling FFmpeg for another OS or CPU architecture?